go/ct/spc/specification.go

```diff
@@ -1557,7 +1554,6 @@ func getAllRules() []Rule {
 		pushes:    1,
 		conditions: []Condition{
 			Eq(ReadOnly(), false),
-			RevisionBounds(MinRevision, R11_Paris),
```

Comment:
why remove them instead of making them `AnyKnownRevision()` ?

Reply:
This constrained is already added by the `rulesFor` function.